1.9/api/access.md

```diff
@@ -0,0 +1,40 @@
+---
+post_title: Cluster Access
+nav_title: Cluster Access
```

Comment:
not technically. 
Does the wordpress integration add them back in for the enterprise ones?